### DISCUSSION TITLE
Fabric Textbox and IText cursor and boundary box is not in the perfect position

### DISCUSSION THREAD

**Discussion by Lukuzushiki** [fabricjs/fabric.js#10156] [General]
Hello. Im using "fabric": "^6.0.2" here. The problem i got is somehow the text pointer is not in the expected position here.
![image](https://github.com/user-attachments/assets/c7a1e715-de3c-4409-b2bd-bc5382360193)

This problem causing when i print to PDF i got the cut off on the text. And also when we trying to edit the text the pointer not in the right position so it makes confuse when typing and delete the text

**Comment by asturur** [comment]
Please check the other issues of cursor not positioned correctly or generic custom font issues before opening new ones
